docs/src/main/asciidoc/spring-cloud-square.adoc

```diff
@@ -0,0 +1,163 @@
+= Spring Cloud Square
+include::_attributes.adoc[]
+
+include::intro.adoc[]
+
+NOTE: Spring Cloud is released under the non-restrictive Apache 2.0 license.
+If you would like to contribute to this section of the documentation or if you find an error, you can find the source code and issue trackers for the project at {docslink}[github].
+
+== Quick Start
+
+include::quickstart.adoc[leveloffset=+1]
+
+Spring Cloud Square The project provides Spring Cloud LoadBalancer integration for https://square.github.io/okhttp/[OkHttpClient] and https://square.github.io/retrofit/[Retrofit], as well as a WebClient-backed Retrofit clients.
+
+== OkHttpClient Spring Cloud LoadBalancer integration
+
+An interceptor is added to the `OkHttpClient` created via autoconfiguration which resolves the scheme, host and port from Spring Cloud LoadBalancer and rewrites the url.
+
+This functionality can be accessed by annotating `OkHttpClient.Builder` beans with `@LoadBalanced`:
+
+[source,java]
+----
+@Configuration
+class OkHttpClientConfig{
+@Bean
+@LoadBalanced
+public OkHttpClient.Builder okHttpClientBuilder() {
+    return new OkHttpClient.Builder();
+    }
+}
+----
+
+Then you can use the `serviceId` or virtual hostname rather than an actual `host:port` in your requests - SC LoadBalancer will resolve it by selecting one of available service instances:
+
+[source,java]
+----
+Request request = new Request.Builder()
+                        .url("http://serviceId/hello").build();
+Response response = builder.build().newCall(request).execute();
+----
+
+You can also https://docs.spring.io/spring-cloud-commons/docs/current/reference/html/#custom-loadbalancer-configuration[use the `@LoadBalancerClient` and `@LoadBalancerClients` annotations to pass custom configuration] to load-balanced OkHttpClient instances.
+
+ This integration supports all the LoadBalancer features. You can read more about them in the https://docs.spring.io/spring-cloud-commons/docs/current/reference/html/#spring-cloud-loadbalancer[Spring Cloud Commons documentation].
+
+You can also disable OkHttpClient load-balancing via properties, by setting the value of `okhttp.loadbalancer.enabled` to `false`.
+
+== Retrofit integration
+
+We provide Spring Boot and Spring Cloud LoadBalancer integration for https://square.github.io/okhttp/[Retrofit], which is a declarative HTTP client.
+
+In order to enable and instantiate Retrofit clients, use the `@EnableRetrofitClients` annotation over a Spring `@Configuration` class:
+
+[source,java]
+----
+@EnableRetrofitClients(clients = TestClient.class, defaultConfiguration = LoggingRetrofitConfig.class)
+@Configuration
+class AppConfiguration{
+
+}
+----
+
+It scans for interfaces that declare they are clients (via `@RetrofitClient`. You can further control the scope of the scan by specifying packages or client names in the annotation parameters. You can also use the `defaultConfiguration` parameter to specify the default config class for all your Retrofit clients.
```

Comment:
Change "(via `@RetrofitClient`" to "(through `@RetrofitClient`)"

(Note the closing parenthesis.)